packages/flutter/lib/src/material/checkbox.dart

```diff
@@ -193,17 +193,10 @@ class _CheckboxState extends State<Checkbox> with TickerProviderStateMixin {
           break;
       }
     }
-    final RenderObject renderObject = node.context.findRenderObject();
+    final RenderObject renderObject = primaryFocus.context.findRenderObject();
```

Comment:
How is it guaranteed that this Widget has primary focus when the action is activated?